.circleci/config.yml

```diff
@@ -3,22 +3,22 @@ version: 2.1
 orbs:
   prodsec: snyk/prodsec-orb@1
 
-executors:
-  linux:
-    machine:
-      image: ubuntu-2204:2023.10.1
```

Comment:
Realized VM is overkill here and it also didn't have the right Go version installed.